Libraries/src/Amazon.Lambda.LexV2Events/Amazon.Lambda.LexV2Events.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\buildtools\common.props" />
+
+  <PropertyGroup>
+    <Description>Amazon Lambda .NET Core support - Amazon Lex V2 package.</Description>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Can we just target `netcoreapp3.1` and then we can get rid of all the `#ifdef` and just always have the system.text.json attribute. Since the lowest version of .NET anybody would be using this is .NET Core 3.1 I don't think we need to maintain that extra logic.

Reply:
Done.

Also made the following changes:
- Removed `[DataContract]` attribute at class level.
- Removed `[DataMember]` attribute and corresponding `#if NETCOREAPP3_1`/`#endif` statements.
- Modified event tests to remove `[InlineData(typeof(JsonSerializer))]` attribute and conditional `#if NETCOREAPP3_1_OR_GREATER` start/end statements.

Please review if this is correct.

Reply:
> You still need the `[DataContract]` and `[DataMember(Name = "xx")]` for anybody that is using the Newtonsoft serializer. I just wanted to get rid of the `#if NETCOREAPP_3_1` all over the place.

Fixed. Please review.